### PR TITLE
Adds a StoreJSON function to the LocalVectoreStore, with unit-test

### PR DESCRIPTION
**Pull request by deckarep**
Hello @RussellLuo ,

Per issue: #1, here is my contribution to include a `StoreJSON` function on the LocalVectoreStore which allows the serialization of the data to the file-system for starters. Let me know if this looks good or you'd like any changes.

Thanks for the input and allowing this contribution!

Cheers,

-Deckarep

### CODE REVIEW

File: vectorstore.go [hunk 1]
```diff
@@ -39,6 +40,29 @@ func (vs *LocalVectorStore) LoadJSON(ctx context.Context, filename string) error
...
 }
 
+// StoreJSON will serialize the `LocalVectorStore` to disk based on the provided filename.
+func (vs *LocalVectorStore) StoreJSON(filename string) error {
+	var chunks []*Chunk
+
+	for _, chunk := range vs.chunks {
+		for _, c := range chunk {
+			chunks = append(chunks, c)
+		}
```
Comment (RussellLuo) [inline vectorstore.go:50]: This is fine. How about `chunks = append(chunks, chunk...)`?

File: vectorstore_test.go [hunk 2]
```diff
@@ -18,6 +18,11 @@ func TestLocalVectorStore_Query(t *testing.T) {
 		t.Fatalf("err: %v\n", err)
 	}
 
+	// Serialize back to disk.
+	if err := store.StoreJSON("testdata/saved_olympics_section.json"); err != nil {
+		t.Fatalf("err: %v\n", err)
+	}
```
Comment (RussellLuo) [inline vectorstore_test.go:24]: Strictly speaking, neither `LoadJSON` nor `StoreJSON` has its own unit test currently.

I guess a better way is to add two unit tests for them. But I can add the tests later if you don't have bandwidth :)

Comment (deckarep) [reply vectorstore_test.go]: @RussellLuo - ok, I was piggy-backing off of this existing unit-test but I'll remove this and let you unit-test the Load/Save how you see fit. Thanks!

File: vectorstore.go (hunk 1 above)
Comment (deckarep) [reply vectorstore.go]: Great suggestion.

**Comment by deckarep** [comment]
Please see latest changes.

**Comment by RussellLuo** [comment]
Looks great, I'm going to merge this PR. Thank you very much for the contribution!

**Pull request merged**